src/main/java/de/cronn/diff/Main.java

```diff
@@ -28,7 +28,7 @@
 
     private static String workingDir = FileHelper.getWorkingDir();
 
-	public static Long MAX_ALLOWED_FILESIZE_DIFFERENCE = 5000000l;
+	public static Long MAX_ALLOWED_FILESIZE_DIFFERENCE = 500000l;
```

Comment:
This commit should be squashed into the previous commit (it even has the same commit message).

Reply:
done.